### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'tmp\\ocr.txt'

### ISSUE THREAD

**Issue by joe159119** [will7200/Yugioh-bot#61]
When I running bot, it can find npc and click once.
And then bot will get stuck and show the error msg

FileNotFoundError: [Errno 2] No such file or directory: 'tmp\\ocr.txt'

I play on Steam and use install.py to setup bot.
All requirement are installed.
 
https://imgur.com/a/mIRG5D8

**Comment by joe159119** [comment]
I fixed thanks

**Comment by will7200** [comment]
Tess not installed in correct location?

**Comment by joe159119** [comment]
Yes, I install Tess manual and forget to copy Tess to correct location. Thank you for reply!
